src/client/components/map/index.ts

```diff
@@ -97,9 +103,12 @@ export function getChoroplethStops(metricKey: string, threshold?: number) {
   }
 }
 
-export function getChoroplethLabels(metricKey: string) {
+export function getChoroplethLabels(metricKey: string, populationDeviation?: number) {
   const compactnessLabels = ["0-30%", "30-40%", "40-50%", "50-60%", ">60%"];
-  const steps = getChoroplethStops(metricKey);
+  const steps =
+    populationDeviation !== undefined
+      ? getChoroplethStops(metricKey, populationDeviation)
+      : getChoroplethStops(metricKey);
```

Comment:
This can be simplified:
```suggestion
  const steps = getChoroplethStops(metricKey, populationDeviation);
```